source/guides/network/connecting-to-mongodb.rst

```diff
@@ -42,7 +42,14 @@ And then use this file when configuring ``symbol-bootstrap``, for example:
 
     symbol-bootstrap start -p testnet -a dual -c custom_parameters.yml
 
-This will make the database's TCP port ``27017`` accessible from the host so be careful, and only use this feature **for development purposes**.
+Previously configured nodes may need to be stopped and restarted with the --upgrade -d argument in order to process the updated custom yml file:
```

Comment:
```suggestion
Previously configured nodes may need to be stopped and restarted with the ``--upgrade -d`` argument in order to process the updated custom yml file:
```

Things that the reader should copy verbatim, like file names or arguments, should be between double backticks.

Also, when used for the first time, I like to use the long version of parameters. Right now I don't remember what ``-d`` does! :D

Reply:
Oh drats, may have messed up with the review - still getting familiar with Github.. We can remove -d entirely, but should ALSO consider recommending that the user backs up their target folder before using the --upgrade command.

`  --upgrade
      It regenerates the configuration reusing the previous keys. Use this flag when upgrading the version of bootstrap to
      keep your node up to date without dropping the local data. **Backup the target folder before upgrading.**`